db/migrate/20200506224301_add_adoption_status_to_pets.rb

```diff
@@ -0,0 +1,5 @@
+class AddAdoptionStatusToPets < ActiveRecord::Migration[5.1]
+  def change
+    add_column :pets, :adoption_status, :string
```

Comment:
Wasn't sure if adoption status should be a string or a boolean. I went with string in case there are more than two options.